js/layout-javascript/agenda-code.js

```diff
@@ -199,6 +199,9 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
 
+      $(this).parents('.new-agenda-search-filter-overlay').removeClass('display');
+      $('.section-top-wrapper, .agenda-cards-wrapper, .dynamic-list-add-item').removeClass('hidden');
```

Comment:
Ues `_this.$container` to restrict selection within the widget instance.

Reply:
@tonytlwu Fixed